profile/README.md

```diff
@@ -12,6 +12,7 @@ Included in the TrustyAI core library are several tools, which include:
 - Fairness metrics
 - Drift metrics
 - Language performance metrics
+- Generative text model evaluation and benchmark 
```

Comment:
Can we keep the tense consistent, e.g.:
- Generative text model evaluations and benchmarks

Reply:
Thanks @RobGeada, fixed